src/lib/storage/column_iterables.hpp

```diff
@@ -80,6 +80,45 @@ class ColumnIterable {
     });
   }
 
+  /**
+   * Materialize all values in this Iterable.
+   * Fallback implementation. Derived Iterables can implement their own, faster version.
```

Comment:
Should we add a PerformanceWarning for this?

Reply:
No, it's probably the best possible implementation for anything but the ValueColumn